Makefile

```diff
@@ -1,6 +1,6 @@
 DEFAULT_GOAL := help
 
-NS ?= mozillaparsys
+NS ?= mozilla
```

Comment:
I can probably undo this. I think once we've got CircleCI building and pushing images, we can remove the tag and release rules.